packages/eddsa-pcd/src/util/util.ts

```diff
@@ -1,5 +1,5 @@
-import * as ed from "noble-ed25519";
+import { getRandomValues } from "@pcd/util";
 
 export function newEdDSAPrivateKey(): string {
-  return Buffer.from(ed.utils.randomPrivateKey()).toString("hex");
+  return Buffer.from(getRandomValues()).toString("hex");
```

Comment:
the `util` package has a function `toHexString` that you could use here

Reply:
Yes, I've seen them. I was also thinking that this function can also be part of the util package. What do you think?